Create thegoodcloud.md (#29)
Deltares/Ribasim-NL: docs/thegoodcloud.md

# Connecting The Good Cloud

> ## modules

Reviewer: ```suggestion
## Modules
```

Just `os`, `pathlib`, `requests` are required to get started
```

> Import os

Reviewer: ```suggestion
import os
```

from pathlib import Path
import requests
```

## global variables

> We define a few global variables, `RIBASIM_NL_CLOUD_PASS` is to be supplied as an os environmnt variable. You need to get one from Deltares first.

Reviewer: ```suggestion
We define a few global variables, `RIBASIM_NL_CLOUD_PASS` is to be supplied as an OOS environment variable.
You need to get one from Deltares first.
```

```

> #
> RIBASIM_NL_CLOUD_PASS = os.getenv("RIBASIM_NL_CLOUD_PASS")

Reviewer: ```suggestion
RIBASIM_NL_CLOUD_PASS = os.getenv("RIBASIM_NL_CLOUD_PASS")
```

RIBASIM_NL_CLOUD_USER = "nhi_api"
WEBDAV_URL = "https://deltares.thegood.cloud/remote.php/dav"
BASE_URL = f"{WEBDAV_URL}/files/{RIBASIM_NL_CLOUD_USER}/D-HYDRO modeldata"

try:
    assert RIBASIM_NL_CLOUD_PASS is not None
except AssertionError:
    raise ValueError(

>         f"Put RIBASIM_NL_CLOUD_PASS in your os environment first."

Reviewer: ```suggestion
        "Put RIBASIM_NL_CLOUD_PASS in your OS environment variables first."
```

    )
```

> ## local-path and remote url

Reviewer: ```suggestion
## Local path and remote URL
```
